### PR TITLE
Add PyQL to README as part of tools built using GitQL SDK

### PR DESCRIPTION
**Pull request by galenseilis**
It may be too soon to accept this PR, but I am just adding it here for later as a change that is unlikely to have merge issues with other future changes.

### CODE REVIEW

**Review by AmrDeveloper — COMMENTED** [review]
Can you please add it too 

https://github.com/AmrDeveloper/GQL/blob/master/docs/sdk/index.md

And also add LLQL to the website docs

https://github.com/AmrDeveloper/LLQL

File: docs/sdk/index.md [hunk 1]
```diff
@@ -47,4 +47,7 @@ To run SQL query on C/C++ Code.
 - [FileQL](https://github.com/AmrDeveloper/FileQL):
 To run SQL query on the file system.
 
-Feel free to add your product too, everyone is welcome to join.
\ No newline at end of file
+- [PyQL](https://github.com/AmrDeveloper/PyQL):
+To run SQL query on Python Code.
+
```
Comment (AmrDeveloper) [inline docs/sdk/index.md:52]: Can you please add

`- [LLQL](https://github.com/amrdeveloper/llql): Tool to run SQL query on LLVM IR/BC and perform Pattern matching on instructions.`

**Review by AmrDeveloper — CHANGES_REQUESTED** [review]

File: docs/sdk/index.md (hunk 1 above)
Comment (galenseilis) [reply docs/sdk/index.md:52]: Absolutely, one moment. :)

Comment (galenseilis) [reply docs/sdk/index.md:55]: ✅ Done!

**Review by AmrDeveloper — APPROVED** [review]

File: docs/sdk/index.md (hunk 1 above)
Comment (AmrDeveloper) [reply docs/sdk/index.md:55]: Thank you

**Pull request merged**